compaction.go

```diff
@@ -1654,15 +1607,12 @@ func (d *DB) flush1() (bytesFlushed uint64, err error) {
 		}
 	}
 
+	bytesFlushed = c.bytesIterated
```

Comment:
@sumeerbhola Is the `InternalIntervalMetrics.Flush.WriteThroughput` indeed intended to measure the bytes iterated over in the flushables instead of the bytes written to disk? I had assumed the `bytesFlushed` return value was the latter and was surprised by this.